Documentation/k8s-pre-reqs.md

```diff
@@ -52,6 +52,17 @@ Rancher provides an easy way to configure Kubelet. This flag can be provided to
 
 To configure Flexvolume in Rancher, specify this Kubelet flag as shown below:
 
+```bash
+--volume-plugin-dir=/var/lib/kubelet/volumeplugins
+```
+
+Restart Kubelet in order for this change to take effect.
```

Comment:
Restarting the kubelet is only required on K8S 1.7 or older

Reply:
All versions of Kubernetes require a Kubelet restart when you change it's arguments.  You're thinking of the runtime install of the flexvolume driver, which requires a kubelet restart only on 1.7 and older.

Reply:
oh, good clarification